### ISSUE TITLE
[dradis-nessus] Blank lines that contain spaces break code blocks and cause export errors

### ISSUE THREAD

**Issue by etdsoft** [dradis/dradis-ce#102]
If the Nessus XML contains blank lines that actually contain a string of blank spaces, the code blocks will break within Dradis and cause errors on export. 

The `<plugin_output>` tag of the Nessus XML may contain blocks of code. If these blocks contain lines that appear blank but are actually strings of empty spaces, these blank lines will import into Dradis and break the code blocks on export, causing errors like the one pictured below: 

<img width="427" alt="screen shot 2016-09-07 at 3 53 49 pm" src="https://cloud.githubusercontent.com/assets/11186655/18328179/5b8cdd2e-7513-11e6-82de-57588a8ee68c.png">

The code block below will generate the error (above) when exported out of Dradis. If the lines that appear blank (e.g. the line before `href="users [...]`) are deleted and replaced with regular line breaks, the export error will disappear. 

```
#[Description]#
bc.. 
-------- output --------
<td><a class="button" 

        href="users [...]
-------- vs --------
<td><a class="button" 

         href="dashboard [...]

        href="true [...]
------------------------

+ The parameter:

-------- output --------
<script type="text/javascript"></scrip [...]
                createCookie('userName', '', 365);
                createCookie('userLocale','en', 365);

                document.location.href = "/dashboard [...]
-------- vs --------
<script type="text/javascript"></scrip [...]
                createCookie('userName', '', 365);

                      document.location.href = "/systeminfo[...]
                    </script>The User [...]
------------------------

```

Bug initially reported by Keith

**Comment by rachkor** [comment]
More details on this (closed) issue: https://github.com/securityroots/dradispro-tracker/issues/310

**Comment by etdsoft** [comment]
Fixed ships in next release.

**Comment by rachkor** [comment]
Reopening as we had another instance of this exact error showing up in v3.0.0/v3.1.0.

**Comment by aapomm** [comment]
Fixed for v3.2.0
